content/en/logs/log_collection/java.md

```diff
@@ -241,6 +241,7 @@ writer.field.dd.env        = {context: dd.env}
 {{< /tabs >}}
 
 #### Inject trace IDs into your logs
+<div class="alert alert-info">Starting in Datadog Agent version 0.74.0, the Java tracer automatically injects trace correlation identifiers into JSON formatted logs. For earlier versions, enable automatic injection in the Java tracer by adding <code>dd.logs.injection=true</code> as a system property, or through the environment variable <code>DD_LOGS_INJECTION=true</code>. Full configuration details can be found on the Java tracer configuration page.</div>
```

Comment:
```suggestion
<div class="alert alert-info">Starting with Datadog Agent version 0.74.0, the Java tracer automatically injects trace correlation identifiers into JSON-formatted logs. For earlier versions, enable automatic injection in the Java tracer by adding <code>dd.logs.injection=true</code> as a system property, or through the environment variable <code>DD_LOGS_INJECTION=true</code>. Full configuration details can be found on the Java tracer configuration page.</div>
```

Reply:
> Full configuration details can be found on the Java tracer configuration page

I recommend adding a link to this page. (I'm not sure if this is a page in the docs or in the UI.)